frontend/views/pages/PayGroup.vue

```diff
@@ -1,122 +1,128 @@
 <template lang="pug">
-page(pageTestName='payGroupPage' pageTestHeaderName='payGroupTitle')
+page(
+  pageTestName='payGroupPage'
+  pageTestHeaderName='payGroupTitle'
+)
   template(#title='') Pay Group
 
-  template(#sidebar='')
-    h2.subtitle
-      i18n Payments Sent
-
-    h3
-      i18n(
-        :args='{ sent: paymentSummary.sent, total: fakeStore.users.length }'
-      ) {sent} of {total}&nbsp;
-
-      tooltip(v-if='paymentSummary.hasWarning')
-        i.icon-exclamation-triangle
-
-        template(slot='tooltip')
-          strong Payment Declined
-          i18n(tag='p')
-            | Someone didn&rsquo;t confirm your payment. Please mark as payed only when it&rsquo;s done.
-
-    h2.subtitle
-      i18n Payments Confirmed
-
-    h3
-      i18n(:args='{ sent: paymentSummary.confirmed, total: fakeStore.users.length }')
-        | {sent} of {total}
-
-    h2.subtitle
-      i18n Amount Sent
-
-    h3(:class="{'has-text-success': paymentAllDone}")
-      i18n(
-        v-if='paymentAllDone'
-        :args='{ currency: fakeStore.currency, amountTotal: paymentSummary.amountTotal }'
-      ) All {currency}{amountTotal}
-
-      i18n(
-        v-else=''
-        :args='{ currency: fakeStore.currency, amoutPayed:paymentSummary.amoutPayed, amountTotal: paymentSummary.amountTotal }'
-      ) {currency}{amoutPayed} of {currency}{amountTotal}
-
-      progress-bar(
-        :primary='paymentProgress.sent'
-        :secondary='paymentProgress.confirmed'
+  template(#sidebar=''
```

Comment:
Right above this, on line 6, can you change it to be:

```js
template(#title='') {{ L('Pay Group') }}
```

Reply:
But `page.vue` already puts i18n in the titles, if I do this here, I would need to do this in all files

Reply:
@pieer what do you think? If you believe it's good to go, feel free to merge this!

Reply:
So since you fixed this I think it's GTM, unless you want to fix [this span/div issue I noticed](https://github.com/okTurtles/group-income-simple/pull/650/files#r321778773).

Re changing it in "all files", not sure what you mean, but yes, this is how all files that use `Page.vue` need to handle it. Having a look at `Page.vue` you'll see that it just has a `slot` there, and even if it didn't, again, the localization string wrappers only ever take string literals, they never take variables.

Reply:
It's a slot, but the slot is inside i18n.

```
i18n
  slot(name='title')
```

So... if I understood correctly, i118n shouldn't be there and need to be removed

Reply:
If you see other file, for example `Contributions.vue`, we don't use i18n neither. that's what I mean by "chaning all the files" that consume `Page.vue`.

```
// Contributions.vue

template(#title='') Contributions
```

Reply:
Well.. it was a 2 minutes fix. It's all fixed and pushed. If it's okay for you, feel free to merge this.

Reply:
@sandrina-p yes, it should be fixed in all of them, and I see you've just pushed a commit that does that, great job! 👍